src/sentences/wnc.rs

```diff
@@ -0,0 +1,104 @@
+use super::utils::array_string;
+use crate::{
+    parse::{NmeaSentence, TEXT_PARAMETER_MAX_LEN},
+    Error, SentenceType,
+};
```

Comment:
also a good way to organize imports is to have:

1. std/core
2. external crate
3. crate and super imports

https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#stdexternalcrate

No need to change it though, just letting you know.